pkg/app/piped/driftdetector/detector.go

```diff
@@ -59,6 +59,7 @@ type secretDecrypter interface {
 
 type Detector interface {
 	Run(ctx context.Context) error
+	ReportApplicationSyncState(ctx context.Context, appID string, state model.ApplicationSyncState) error
```

Comment:
Why do we need to add this function?

Reply:
As you say, it's not currently in use. Hence I removed it. TYVM:)

Reply:
Thank you.